Source/WebKit/Shared/RemoteLayerTree/RemoteLayerTreePropertyApplier.mm

```diff
@@ -297,6 +299,11 @@ static void applyCommonPropertiesToLayer(CALayer *layer, const RemoteLayerTreeTr
     }
 #endif
 #endif
+
+    if (properties.changedProperties & LayerChange::VideoGravityChanged) {
+        if ([layer respondsToSelector:@selector(setVideoGravity:)])
+            [(WebAVPlayerLayer*)layer setVideoGravity:convertMediaPlayerToAVLayerVideoGravity(properties.videoGravity)];
```

Comment:
Is it worth asserting `[layer isKindOfClass:[WebAVPlayerLayer  class]]`?

Reply:
Maybe? A lot of classes respond to -setVideoGravity:. I almost wish there was an explicit protocol we could check for.